app/classes/Transvision/ShowResults.php

```diff
@@ -322,6 +323,12 @@ public static function resultsTable($search_object, $search_results, $page)
             */
             $transliterate = $locale2 == 'sr' && ! $extra_locale && $target_string && $target_string != '@@missing@@';
 
+            if ($current_repo === 'aurora' && ShowResults::isPresentInPootle($locale2)) {
+                $link = "https://mozilla.locamotion.org/$locale2/firefox/translate/$entityPath.po?search=$strings[1]&sfields=source&sfields=target";
```

Comment:
Let's use a better variable name here, e.g. edit_link
Also,`"{$variable_name"` in strings